src/frontend/App.css

```diff
@@ -37,6 +37,7 @@ body {
 }
 
 .App .content {
+  container: main-content / inline-size;
```

Comment:
using container instead of media queries to have better control since the sidebar can be resized and that changes `main.content` size